src/Components/Album/Album.module.css

```diff
@@ -0,0 +1,32 @@
+.style.container {
```

Comment:
Great idea to divide the css into different files. Makes it much easier to read the code!